src/core/contrib.js

```diff
@@ -17,13 +17,17 @@ function prop(prop) {
   return o => o[prop];
 }
 const nameProp = prop("name");
-const urlProp = prop("url");
 
-function findUserURLs(...thingsWithUsers) {
+function URLByUser(conf, user) {
+  const { githubAPIBase } = conf;
+  return new URL(user.url, githubAPIBase).href;
+}
+
+function findUserURLs(conf, ...thingsWithUsers) {
   const usersURLs = thingsWithUsers
     .reduce(flatten, [])
     .filter(thing => thing && thing.user)
-    .map(({ user }) => user.url);
+    .map(({ user }) => URLByUser(conf, user));
```

Comment:
I have copied the logic directly into the two spots where it's used. I originally decided to split this into its own function because I saw that `nameProp` was, and it is trivial too.